README.md

```diff
@@ -341,6 +343,80 @@ Audited.config do |config|
 end
 ```
 
+## S3 Support
+
+Audited now supports storing audits to Amazon S3 instead of relying on Active Record.
+The motivation behind this is to allow large projects with lots of audits to be
+stored somewhere outside of a traditional SQL database. This provides a better outlet
+for data warehousing and decouples vast audit tables from the primary database.
+
+### Setup:
+
+```ruby
+Audited.config do |config|
+  config.storage_mechanism = :s3
+  config.storage_options = {
+    bucket_name: ENV['S3_BUCKET_NAME'],
+    s3_key_prefix: "#{Rails.env}/#{ENV['S3_PREFIX']}",
+    region: ENV['AWS_REGION'],
+    access_key_id: ENV['AWS_ACCESS_KEY_ID'],
+    secret_access_key: ENV['AWS_SECRET_ACCESS_KEY'],
+    partition: true,
+    unpartitioned_types: ['Company', 'Employee']
+  }
+end
+```
+
+### Partitioning
+
+Because S3 directories can get VERY large if processing lots of audits,
+partitioning is supported. It can be activated by adding the "partition" option
+into the storage_options config as seen above in the Setup section.
+
```

Comment:
Should probably update this to reflect the change in partitioning behavior.  Otherwise, I think it looks good.